Lambdas/AutoFaqs/S3AutoFaqsRepository.swift

```diff
@@ -53,10 +56,13 @@ public struct S3AutoFaqsRepository {
         do {
             return try decoder.decode([String: String].self, from: body)
         } catch {
-            logger.error("Cannot find any data in the bucket", metadata: [
-                "response-body": .string(String(buffer: body)),
-                "error": "\(error)"
-            ])
+            logger.error(
+                "Cannot find any data in the bucket",
+                metadata: [
+                    "response-body": .string(String(buffer: body)),
+                    "error": "\(error)",
+                ]
+            )
```

Comment:
I am really, really, _really_ not a fan of all these extra newlines. Would setting `lineBreakBeforeEachGenericRequirement` and `lineBreakBeforeEachArgument` to `false` make it ease up on this?

Reply:
I did try some combinations of disabling the "lineBreak" settings, but didn't try just disabling all of them at once.